docs/overview.md

```diff
@@ -0,0 +1,34 @@
+# Overview
+
+This document presents a high level overview of the flow of information and steps taken to establish a support tunnel, in technical detail. It is intended to show at a high level that this implementation is securely designed.
```

Comment:
I would suggest describing the overall intent of the support tunnel: Something that only a user can initiate that doesn't expose them to any additional security risks.